WcaOnRails/db/migrate/20160504230105_add_results_posted_to_competitions.rb

```diff
@@ -0,0 +1,14 @@
+class AddResultsPostedToCompetitions < ActiveRecord::Migration
+  def up
+    add_column :Competitions, :results_posted_at, :datetime
+    execute <<-SQL
+      UPDATE Competitions
+      SET results_posted_at = (SELECT MIN(updated_at) FROM Results where competitionId = Competitions.id)
+      WHERE results_posted_at is NULL AND id IN (SELECT competitionId FROM Results)
+    SQL
+  end
```

Comment:
Before migration: http://pastebin.com/whQ1vTGX

After migration: http://pastebin.com/gtEsjaSH

Using MIN() here makes more sense since results are sometimes changed after posted.